CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# HEAD
+# 5.0.1
```

Comment:
Technically, I think this is a patch bump. If we want a minor bump, it should be 5.1.0. Minor makes sense, as it's an addition, not a fix to existing code.

Reply:
Good call